### PR TITLE
Change name of Proof of Service Letter

### PR DESCRIPTION
**Pull request by annekainicUSDS**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4944

### CODE REVIEW

File: src/js/letters/containers/LetterList.jsx [hunk 1]
```diff
@@ -25,6 +25,8 @@ export class LetterList extends React.Component {
             Inquiry Routing & Information System (IRIS)</a>. VA will respond within 5 business days.
           </p>
         );
+      } else if (letter.letterType === LETTER_TYPES.proofOfService) {
+        letterTitle = 'Proof of Service Card';
```
Comment (markgreenburg) [inline src/js/letters/containers/LetterList.jsx]: Do we need to define content like we define content in the `else`?

Comment (annekainicUSDS) [reply src/js/letters/containers/LetterList.jsx]: oops! good catch, will fix that

Comment (annekainicUSDS) [reply src/js/letters/containers/LetterList.jsx]: fixed!

**Review by markgreenburg — APPROVED** [review]
:shipit:

**Pull request merged**